technische-universitat-dresden-erziehungswissenschaften-author-date.csl

```diff
@@ -0,0 +1,251 @@
+  <info>
+    <title>Technische Universität Dresden - Erziehungswissenschaften (author-date)</title>
+    <title-short>TUD Erziehungswissenschaften (author-date)</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-erziehungswissenschaften-author-date</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-erziehungswissenschaften-author-date" rel="self"/>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-author-date" rel="template"/>
+    <link href="https://tu-dresden.de/gsw/ew/ressourcen/dateien/studium/pdf/broschuere_booklet.pdf?lang=de" rel="documentation"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>http://www.slub-dresden.de</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="psychology"/>
+    <summary>Zitierstil entsprechend den Zitierrichtlinien der Fakultät Erziehungswissenschaften, Technische Universität Dresden.</summary>
+    <updated>2016-07-19T16:23:57+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="available at">URL</term>
+      <term name="accessed">Download vom</term>
+      <term name="issue">Heft</term>
+      <term name="volume" form="short">Jg.</term>
+      <term name="et-al"> [u.&#8239;a.]</term>
+      <term name="anonymous" form="short">o.&#8239;V.</term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="et-al"> [et al.]</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" font-variant="normal" delimiter=", ">
+      <name font-variant="normal" delimiter="; " et-al-min="3" et-al-use-first="3" initialize="false" name-as-sort-order="all"/>
+      <label form="short" font-variant="normal" prefix=" (" suffix="):"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" strip-periods="false"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name font-variant="normal" delimiter="; " delimiter-precedes-last="never" initialize="false" initialize-with="." name-as-sort-order="all"/>
+      <et-al font-variant="normal" font-weight="normal" vertical-align="baseline"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter="; " delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" initialize-with=". "/>
+      <et-al/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
```

Comment:
the substitute list should amost certainly be the same for author and author-short: otherwise in-text citation and bibliography will not correspond.